### PR TITLE
font cleanup

### PR DESCRIPTION
**Pull request by jtrein**
* no more external Google CDN requests
* fonts load locally via webpack
* Material Icons font was not needed - dead code - as Material UI React loads them

### CODE REVIEW

File: src/fonts/fonts.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/* Roboto */
+@font-face {
+  font-family: 'Roboto';
```
Comment (jtrein) [inline src/fonts/fonts.css]: The font-family name should be the same across all 3

File: index.html [hunk 2]
```diff
@@ -4,8 +4,6 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width">
     <title>Universal Drug Codes</title>
-    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet" />
```
Comment (jtrein) [inline index.html]: Freaking no more render-blocking network requests

**Comment by Chris-Petty** [comment]
[build ci]

**Comment by Chris-Petty** [comment]
Miss clicked haha

**Pull request merged**